src/shared/lib/GenericAssayUtils/MutationalSignaturesUtils.tsx

```diff
@@ -188,10 +188,17 @@ export function retrieveMutationalSignatureVersionFromData(
         })
     );
     if (uniqueProfileVersion !== undefined) {
-        return uniqueProfileVersion.includes('v3') &&
+        if (
+            uniqueProfileVersion.includes('v3') &&
             uniqueProfileVersion.includes('v2')
-            ? 'v3'
-            : uniqueProfileVersion[0]!;
+        ) {
+            return 'v3';
+        } else if (uniqueProfileVersion.includes('SBS')) {
```

Comment:
@inodb since we already have a mechanism for selecting default mutation sig profile,  i thought it best to enforce the SBS here rather than monkey with data ordering in a hackish way.  what do you think?

Reply:
yeah that looks good to me 👍